src/cpp/redisserver.cpp

```diff
@@ -36,6 +36,27 @@ using namespace SmartRedis;
 // to prevent it, but this will suffice
 static bool ___srand_seeded = false;
 
+// RedisServer constructor
+RedisServer::RedisServer()
+{
+    _init_integer_from_env(_connection_timeout, _CONN_TIMEOUT_ENV_VAR,
+                           _DEFAULT_CONN_TIMEOUT);
+    _init_integer_from_env(_connection_interval, _CONN_INTERVAL_ENV_VAR,
+                           _DEFAULT_CONN_INTERVAL);
+    _init_integer_from_env(_command_timeout, _CMD_TIMEOUT_ENV_VAR,
+                           _DEFAULT_CMD_TIMEOUT);
+    _init_integer_from_env(_command_interval, _CMD_INTERVAL_ENV_VAR,
+                           _DEFAULT_CMD_INTERVAL);
+
+    _check_runtime_variables();
+
+    _connection_attempts = ( _connection_timeout * 1000 ) /
+                             _connection_interval + 1;
```

Comment:
. remove spaces inside parens -- (_connection_timeout * 1000)
 .  remove line wrap if line length is under 80 characters